test/es_test_case.py

```diff
@@ -36,7 +36,8 @@ class ElasticsearchTestCase(DockerContainerTestCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        es_endpoint = cls._create_container('docker.elastic.co/elasticsearch/elasticsearch:7.10.1',
+        # TODO remove drop commit that put the non-hardened ES image here
+        es_endpoint = cls._create_container('docker.elastic.co/elasticsearch/elasticsearch:7.17.10',
```

Comment:
I bumped the internal version to 2 when merging your PR in the other repo:. This is the image we should be using: https://hub.docker.com/layers/ucscgi/azul-elasticsearch/7.17.10-2/images/sha256-d787b3432b5722a833733047dad6009e493fa4fabc024d3bc7e6e5e54902bbb9?context=repo